### DISCUSSION TITLE
What kind of telemetry (if any) does photoprism collect, if I use a self hosted instance?

### DISCUSSION THREAD

**Discussion by Leonetienne** [photoprism/photoprism#3425] [Q&A]
Let's say, I spin up an instance of Photoprism on my server and use it to share vacation pictures with my family.
Would Photoprism "phone home"? Would Photoprism share ANY data with anyone but me and my family?

Let's say I went into the settings and checked "Disable places", which, as far as I can tell, disables the only legitimate reason to call home.

**Comment by lastzero** [comment]
There isn't any telemetry like for example in Traefik that sends us your config files or usage details. However, PhotoPrism connects to our backend once a day to fetch the latest maps key as we frequently change it to prevent abuse. With the same request, it checks your membership tier to unlock additional features like the admin web UI.

Details on this and the Geocoding API we provide can be found in our privacy policy, user guide, and FAQs:
- https://www.photoprism.app/privacy
- https://docs.photoprism.app/getting-started/troubleshooting/firewall/

**Reply by Leonetienne** [reply]
Thanks!
